lib/src/flagged/flagged_bloc.dart

```diff
@@ -123,11 +120,10 @@ class FlaggedBloc extends Bloc<FlaggedEvent, FlaggedState> {
       }
     }
     messageIds.removeWhere((id) => id == ChatMsg.idDayMarker);
-
     add(
       FlaggedMessagesLoaded(
-          messageIds: messageIds,
-          messageLastUpdateValues: _messageListRepository.getAllLastUpdateValues().reversed.toList(growable: false),
+          messageIds: messageIds.reversed.toList(growable: false),
+          messageLastUpdateValues: _messageListRepository.getLastUpdateValuesForIds(messageIds).reversed.toList(growable: false),
```

Comment:
@phranck Just a small fix. 
We now only transfer the messageLastUpdateValues for the `messageIds` not for all. And both lists are reversed now, to match the ordering in the chat view.